second-gen/packages/swc/.storybook/guides/code_of_conduct.mdx

```diff
@@ -0,0 +1,79 @@
+import { Meta, Title } from '@storybook/blocks';
+
+<Meta title="Adobe Code of Conduct" />
```

Comment:
Went ahead and pulled this over from Spectrum CSS since it's a nice easy drop-in.